arkane/gaussian.py

```diff
@@ -466,6 +473,20 @@ def load_scan_frozen_atoms(self):
         """
         return self._load_scan_specs('F')
 
+    def load_scan_angle(self):
```

Comment:
make these functions private with a leading `_`, or add an equivalent function with a "not implemented error" in all other log classes (probably the first option?)

Reply:
good catch. Added the `_` to these two functions.